package.json

```diff
@@ -8,8 +8,9 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.6",
     "react-scripts": "5.0.0",
-    "web-vitals": "^2.1.4"
+    "redux": "^4.1.2"
```

Comment:
you could remove redux if it's not yet used

Reply:
fixed